pkg/operator/controller/externaldns/controller.go

```diff
@@ -77,6 +81,7 @@ func New(mgr manager.Manager, cfg Config) (controller.Controller, error) {
 // the object match the desired state.
 func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	result := reconcile.Result{}
+	requeueResult := reconcile.Result{RequeueAfter: time.Duration(r.config.RequeuePeriodSeconds) * time.Second, Requeue: true}
```

Comment:
Specifying `Requeue` is superfluous if you specify a non-zero value for `RequeueAfter`.